### PR TITLE
[no_severity] Replace the fixed time limitation with an expanding one

### LINKED ISSUE #1592: Drop the time limitation in triage related tools
https://github.com/mozilla/relman-auto-nag/blob/163d45b62a2e499229afeb11198aa0d96678c059/auto_nag/bzcleaner.py#L417-L427

### PR DESCRIPTION
**Pull request by suhaibmujahid**
Resolves #1592

We expand the time limitation for the oldest bug  ~3 weeks each month.

## Checklist

- [ ] Type annotations added to new functions
- [ ] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results
- [ ] The [`to-be-announced`](https://github.com/mozilla/relman-auto-nag/labels/to-be-announced) tag added if this is worth announcing

### CODE REVIEW

File: auto_nag/scripts/workflow/no_severity.py [hunk 1]
```diff
@@ -236,6 +238,23 @@ def get_bz_params(self, date):
                 }
             )
 
+        # TODO: the following code can be removed in 2023.
+        # https://github.com/mozilla/relman-auto-nag/issues/1592#issuecomment-1201893220
+        # Almost 500 old bugs has no severity set. The intent of the following
```
Comment (marco-c) [inline auto_nag/scripts/workflow/no_severity.py:243]:
```suggestion
        # Almost 500 old bugs have no severity set. The intent of the following
```

File: auto_nag/scripts/workflow/no_severity.py [hunk 2]
```diff
@@ -236,6 +238,23 @@ def get_bz_params(self, date):
...
+        # is to have them triaged in patches where every week we include more
```
Comment (marco-c) [inline auto_nag/scripts/workflow/no_severity.py:244]:
```suggestion
        # is to have them triaged in batches where every week we include more
```

File: auto_nag/scripts/workflow/no_severity.py [hunk 3]
```diff
@@ -236,6 +238,23 @@ def get_bz_params(self, date):
                 }
             )
 
+        # TODO: the following code can be removed in 2023.
```
Comment (marco-c) [inline auto_nag/scripts/workflow/no_severity.py:241]: Could you file an issue to track the removal? Or did you want to use #1592 to track it?

**Review by marco-c — APPROVED** [review]

File: auto_nag/scripts/workflow/no_severity.py (hunk 3 above)
Comment (suhaibmujahid) [reply auto_nag/scripts/workflow/no_severity.py:241]: I filed a new issue: https://github.com/mozilla/relman-auto-nag/issues/1596

**Review by marco-c — APPROVED** [review]

**Pull request merged**